bluesky-social/social-app
# [iPad support] Custom tab navigator for native tablet, with side nav

# Stacked on https://github.com/bluesky-social/social-app/pull/1569

This PR create a custom navigatior for the native tablet layout, mimicking the web tablet layout. However, the web navigator is in a flat structure, whereas on native it needs to maintain a stack navigator for each tab.

![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-04-03 at 13 46 18](https://github.com/bluesky-social/social-app/assets/10959775/f098a53e-c378-4978-a95e-b866c36b1702)

# Test plan

1. Verify that the left nav is working for all tabs
2. Make sure no links are broken, especially to the pages that have been turned into tabs
3. Check that the header doesn't overlap the feed
4. Check that there's no emoji picker in the composer, and other random polish things like that
